core/src/main/contraband/librarymanagement.json

```
@@ -733,6 +733,19 @@
         { "name": "devConnection", "type": "String?",     "default": "None", "since": "0.0.1" }
       ]
     },
+    {
+      "name": "LicenseInfo",
+      "namespace": "sbt.librarymanagement",
+      "target": "Scala",
+      "type": "record",
+      "doc": "Basic license information for a project module",
+      "fields": [
+        { "name": "name",          "type": "String",      "default": "\"\"",                        "since": "2.0.0" },
+        { "name": "url",           "type": "java.net.URL","default": "new java.net.URL(\"http:\")", "since": "2.0.0" },
```

please use `java.net.URI` here as done in  #443 
```suggestion
        { "name": "url",           "type": "java.net.URI","default": "new java.net.URL(\"http:\")", "since": "2.0.0" },
```

---

Changed and pushed.